applications/osb-portal/src/components/workspace/AddResourceForm.tsx

```diff
@@ -25,9 +57,297 @@ function isValidHttpUrl(s: string) {
   return url.protocol === "http:" || url.protocol === "https:";
 }
 
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box>{children}</Box>
+      )}
+
+    </div>
+  );
+}
+
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
200+ lines of styling for a single component are not per se wrong, but don't smell good. Better think at some way to reduce the amount of styling